examples/Remarketing/UploadStoreSalesTransactions.cs

```diff
@@ -283,7 +297,7 @@ public override string Description
         /// <param name="customerId">The Google Ads customer ID for which the call is made.</param>
         /// <param name="offlineUserDataJobResourceName">The resource name of the job to which to
         ///     add transactions.</param>
-        /// <param name="conversionActionId">The ID of a store sales conversion action.</param>
+        /// <param name="conversionActionId">The ID of a store sales conversion action.</param> 
```

Comment:
Please ensure that there is no trailing whitespace on this line.

Reply:
Removed the whitespace